LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/constants/ResetOperation.java

```diff
@@ -0,0 +1,23 @@
+package com.microsoft.identity.labapi.utilities.constants;
+
+import lombok.NonNull;
+
+public enum ResetOperation {
+    MFA(LabConstants.ResetOperation.MFA),
+    PASSWORD(LabConstants.ResetOperation.PASSWORD);
+
+    final String value;
+
+    ResetOperation(final String value) {
+        this.value = value;
+    }
+
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    public static ResetOperation fromName(@NonNull final String name) {
+        return valueOf(ResetOperation.class, name.toUpperCase());
+    }
```

Comment:
I think I provided suggestion above to just use the enum directly in the tests (as opposed to calling toString on them) and then you wouldn't need this